joplin/importer/import_everything.py

```diff
@@ -0,0 +1,128 @@
+import os
+from gql.transport.requests import RequestsHTTPTransport
+from gql import Client
+from gql import gql
+from importer.queries import queries
+import json
+from importer.page_importer import PageImporter
+from pages.base_page.models import JanisBasePage
+
+api_url = 'http://joplin-staging.herokuapp.com/api/graphql'
+page_type_map = {
+    'service page': 'services',
+    'guide page': None,
+    'topic collection page': 'topiccollection',
+    'information page': 'information',
+    'department page': 'department',
+    'form container': 'form',
+    'topic page': 'topic',
+    'official document page': 'official_document',
+    'event page': 'event',
+    'location page': 'location'
+}
+
+def get_jwt_token():
```

Comment:
this is pretty much just test_api_jwt_token copypasta, I don't want to just use the test creds and I don't know where a good place for a shareable version would be

suggestions more than welcome!